docs/DeveloperGuide.md

```diff
@@ -1028,7 +1036,56 @@ testers are expected to do more *exploratory* testing.
 
    4. Invalid commands to try (Error details shown in the status message):
       * Invalid index >= size of elderly list or <= 0: `deleteElderly 5` or `deleteElderly -1`
+      
+### Adding tags to an elderly
 
+1. Add one or more tags to an elderly in NurseyBook
+
+    1. Prerequisites: List all elderlies using the `viewElderly` command. Multiple elderlies in the list. The first elderly in the list has one `diabetes` tag.
```

Comment:
Does it have to be all? `findElderly` and `filter` should be okay as well.

Reply:
Ohh I see, it's just for this particular test case's scenario?

Reply:
Yep this part is just some suggestions for manual testing. So this is a scenario.